drivers/gpu/drm/drm_fb_helper.c

```diff
@@ -1967,6 +1967,15 @@ __drm_fb_helper_initial_config_and_unlock(struct drm_fb_helper *fb_helper,
 	 * register the fbdev emulation instance in kernel_fb_helper_list. */
 	mutex_unlock(&fb_helper->lock);
 
+	id = of_alias_get_highest_id("drm_fb");
```

Comment:
It's a nit pick, but the comment says `drm-fb`.

Reply:
Commit text updated to also say ```drm_fb```